scripts/azure_deployment/arm_aci.py

```diff
@@ -32,7 +32,7 @@ def get_pubkey():
 STARTUP_COMMANDS = {
     "dynamic-agent": lambda args: [
         "apt-get update",
-        "apt-get install -y openssh-server rsync",
+        "apt-get install -y openssh-server rsync sudo",
```

Comment:
I'm surprised this is necessary?

Reply:
Yes, in the golang image, sudo is not included